### DISCUSSION TITLE
installation folder

### DISCUSSION THREAD

**Discussion by Whateverman2012** [nutti/fake-bpy-module#129] [Q&A] [answered]
Hi,
  First, thanks for the stubs !

  I have multiple programming environments and wonder if the fake-module could be installed in a folder instead of directly in site-package ?

  It is hard to see what files are belonging to that module compared to others which are usually using -info and a master folder.

  Regards.

**Comment by nutti** [comment, marked as answer]
@Whateverman2012 

Yes, you can use this package without the installation.
I'm not sure you use VSCode or not.
Below document shows how to use fake-bpy-module without installation via pip.
https://github.com/nutti/fake-bpy-module/blob/master/docs/setup_visual_studio_code.md

**Comment by Whateverman2012** [comment]
Thank !

I didn’t see the pre-generated part.

Regards.
